ssg-rhel7-ds-tailoring.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xccdf:Tailoring xmlns:xccdf="http://checklists.nist.gov/xccdf/1.2" id="xccdf_scap-workbench_tailoring_default">
+  <xccdf:benchmark href="/usr/share/xml/scap/ssg/content/ssg-rhel7-ds.xml"/>
+  <xccdf:version time="2019-11-22T19:09:04">1</xccdf:version>
+  <xccdf:Profile id="xccdf_org.globalnet_profile_stig-rhel7-disa_tailored" extends="xccdf_org.ssgproject.content_profile_stig-rhel7-disa">
+    <xccdf:title xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en-US" override="true">DISA STIG for Red Hat Enterprise Linux 7 [CUSTOMIZED]</xccdf:title>
+    <xccdf:description xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en-US" override="true">**Not applicable to CentOS Linux, included for reference only**
+
+This profile contains configuration checks that align to the
+DISA STIG for Red Hat Enterprise Linux V1R4.
+
+In addition to being applicable to RHEL7, DISA recognizes this
+configuration baseline as applicable to the operating system tier of
+Red Hat technologies that are based off RHEL7, such as:
+
+- Red Hat Enterprise Linux Server
+- Red Hat Enterprise Linux Workstation and Desktop
+- Red Hat Enterprise Linux for HPC
+- Red Hat Storage</xccdf:description>
+    <xccdf:select idref="xccdf_org.ssgproject.content_rule_mount_option_krb_sec_remote_filesystems" selected="false"/>
```

Comment:
It looks like there are more items here than in https://github.com/CivicActions/hailstone/blob/master/ssg-rhel7-ds-justifications.yaml - do we need to update the justifications documentation?
Also, ideally this file wouldn't be committed to git, but we would dynamically generate the list from the justifications.yaml file using a script (e.g. python) as part of the pipeline (in packer, ideally - can leave the justifications file in the image for reference).

Reply:
As discussed in Slack conversation with Owen today, here is the summary for the above request change. We need to leave the tailoring file as it is for now because packer utilizes this file while building an AMI. Also, manually adding remaining items from this tailoring file into justification file to keep both files in sync and should be worked later on task that i have created which will generate tailoring file at run time.